static/js/views/add-thing.js

```diff
@@ -93,7 +93,7 @@ const AddThingScreen = {
 
     API.startPairing(timeout)
       .then((json) => {
-        this.actionUrl = json.pair.href;
+        this.actionUrl = json.href;
```

Comment:
does this take into account the `base` field?

Reply:
That's a good question, but I don't think it actually matters in the case of the gateway's front end because it's not a general purpose WoT Consumer, it only consumes Thing Descriptions exposed by the gateway. All Thing Descriptions exposed by the gateway have the [base set to the root of the host](https://github.com/WebThingsIO/gateway/blob/master/src/models/thing.ts#L638) and all values of `href` are absolute paths. Therefore the action URL will always be an absolute path, relative to the gateway's own host.

The WebThings Cloud implementation I'm working on next _is_ a general purpose WoT Consumer, and so will have to resolve the base URL properly.